DnD-Montreal/session-tome
# Event Report Generation

## Description
Closes #249 
Creates a backpack endpoint that allows admin users to generate a report on a specific Event.
The report contains a list of all characters, their users, and the character's info.

## How to test
Make sure your database is migrated and seeded
Open tinker and run `$event = App\Models\Event::has('sessions')->first()`
Copy the title of that event
Log in as the admin user and head to /admin/event
Enter the title in the search bar
Click the "Generate Report" button
Your browser should attempt to download a csv. 
Check that the csv has character info and run the following command to compare the info
`$event->sessions()->with('characters')->get()`
